101-signalr/azuredeploy.json

```diff
@@ -0,0 +1,61 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "name": {
+      "type": "string",
+      "metadata": {
+        "description": "The name of the SignalR resource to create."
```

Comment:
does this need to be globally unique? if so, add a defaultValue with uniqueString and update description

Reply:
Yes it's globally unique. Updated.